values/values.cassandra.yaml

```diff
@@ -8,6 +8,24 @@ server:
       default:
         driver: "cassandra"
 
+#        faultInjection:
```

Comment:
I leave it here for future references as example on how to configure fault injection.